packages/reporters/dev-server/src/Server.js

```diff
@@ -362,20 +362,11 @@ export default class Server {
       return;
     }
 
-    return serveHandler(
-      req,
-      res,
-      {
-        public: root,
-        cleanUrls: false,
-      },
-      {
-        lstat: path => fs.stat(path),
-        realpath: path => fs.realpath(path),
-        createReadStream: (path, options) => fs.createReadStream(path, options),
-        readdir: path => fs.readdir(path),
```

Comment:
It's not possible to overwrite the FS of server-static? You could use an in-memory FS with Parcel, then it would not work without this

Reply:
I thought the `fs.*` refer to real files.

In-memory FS: `/dev/shm/xxxx`

Reply:
Parcel has other implementations of the fs interface, such as https://github.com/parcel-bundler/parcel/blob/v2/packages/core/fs/src/MemoryFS.js

That's why the `fs` parameter exist in that function

Reply:
In that case, Parcel can fork serve-static (it's small).
 
https://github.com/expressjs/serve-static/blob/master/index.js